apis/spanner/v1beta1/instance_types.go

```diff
@@ -53,7 +53,8 @@ type SpannerInstanceSpec struct {
 	AutoscalingConfig *AutoscalingConfig `json:"autoscalingConfig,omitempty"`
 
 	// Optional. The `Edition` of the current instance.
-	// Currently accepted values are EDITION_UNSPECIFIED, STANDARD, ENTERPRISE, ENTERPRISE_PLUS
+	// Currently accepted values are STANDARD, ENTERPRISE, ENTERPRISE_PLUS
+	// If edition is unspecified, the default value is STANDARD.
```

Comment:
From the doc, it seems that there's a chance this default value could be different than standard
https://cloud.google.com/spanner/docs/editions-overview
How about "If edition is unspecified, Spanner server will use the lowest edition that matches your usage pattern to avoid workload disruption."